stacks/root.yml

```diff
@@ -332,6 +334,49 @@ Resources:
         PlaySecretsVersion: !Ref PlaySecretsVersion
       TemplateURL: !Join ["", ["http://s3", !If [IsUsEast1, "", "-"], !If [IsUsEast1, "", !Ref "AWS::Region"], ".amazonaws.com/", "Fn::ImportValue": !Sub "${InfrastructureStorageStackName}-InfrastructureSourceBucket", "/", !Ref InfrastructureGitCommit, "/stacks/play.prx.org.yml"]]
       TimeoutInMinutes: "5"
+  ##############################################################################
```

Comment:
I've only done a cursory review of these changes so far; focused on the other template for this review